package.json

```diff
@@ -13,15 +13,15 @@
     "prestart": "npm run build:contract && npm run dev:deploy:contract",
     "start": "CONTRACT_NAME=$(cat neardev/dev-account) parcel src/index.html",
     "dev": "nodemon --watch assembly -e ts --exec 'npm run start'",
-    "test": "npm run build:contract && jest test --env=near-shell/test_environment"
+    "test": "npm run build:contract && npm run dev:deploy:contract && jest test"
```

Comment:
`npm run dev:deploy:contract` why is this needed? Tests shouldn't be running on the same contract instance as `dev:deploy:contract` uses.

Reply:
that was my mistake. meant to remove it. just pushed up a commit removing it.